packages/blockchain-api/package.json

```diff
@@ -17,7 +17,7 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
-    "@celo/contractkit": "0.2.0",
+    "@celo/contractkit": "0.2.10-dev",
```

Comment:
blockchain api, notification service, and verification pool can only rely on published versions. A limitation of the AppEngine / Cloud Functions deployment process. Recommend just leaving their CK versions as is unless they need this upgrade. Or upgrade them to the latest published version (not -dev) but would be  good to test them before merging :)